RELEASE.md

```diff
@@ -73,10 +73,15 @@ cd platforms/android && ./gradlew publish closeAndReleaseRepository`
 ```
   
 ### Swift/iOS:
-This should be done automatically when a tag is uploaded however these are the steps to do it manually:
+When a tag is uploaded the `./release_ios.sh` will be called with the `-t <version>` (where `<version>` will be the provided tag).
+This will open a PR against [the swift package repo](https://github.com/matrix-org/matrix-wysiwyg-composer-swift) with the latest changes.
+Remember to manually check and merge the PR that will get opened on the [SWIFTPM Repo PR list](https://github.com/matrix-org/matrix-wysiwyg-composer-swift/pulls), and provide as commit for the merge the same name of the branch/PR.
```

Comment:
This is a bit confusing. Does this mean that the commit name should be the name of the branch? Or that we should check that the right branches/tags are used in the PR?

Reply:
just that the commit name should be the name of the branch (so that the tag will automatically be applied on main when the merge is done).
Any suggestions on how to make this less confusing?

Reply:
Maybe:

"Remember to manually check that the opened PR on the [SWIFTPM Repo](https://github.com/matrix-org/matrix-wysiwyg-composer-swift/pulls) exists and it contains the source branch name as the commit message, then merge it."

Reply:
But I feel in this way doesn't feel clear that we need to manually provide the commit message when merging to match the name of the branch.
What about...
"Remember to manually check the PR that will get opened on the [SWIFTPM Repo PR list](https://github.com/matrix-org/matrix-wysiwyg-composer-swift/pulls), and before merging it provide as commit message the name of the branch/PR."

Reply:
Sounds good to me. I guess it seems a bit confusing because you need to override the commit message.

Reply:
Yeah I know, would be way easier if the workflow could access not the commit message but the commit comment (since by the default is already the branch name), but seems that there is no way to do so